tests/KubernetesClient.Tests/OperatingSystem.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace k8s.Tests
+{
+    [Flags]
+    public enum OperatingSystem
```

Comment:
why not inline this in OperatingSystemFact.cs? I don't think it's used anywhere else.

Reply:
oh, I take it back, I see it in the test too. nevermind.